### DISCUSSION TITLE
how to use optimization tools?

### DISCUSSION THREAD

**Discussion by nadia-am** [spatie/image-optimizer#137] [Q&A] [answered]
I am using this package on my localhost, how to use optimizer tool on windows? how to install them on windows?

**Comment by younixue3** [comment]
did u find the solution? I have the same question, I do the Optimizer but the size did not get optimize

**Reply by nadia-am** [reply]
no i didnt

**Reply by nadia-am** [reply]
actually you should install the tools which mentioned in documentation and the tools could not be install on windows!

**Reply by yellow1912** [reply]
you can use windows linux subsystem if you really want to use this on windows.

**Comment by Tyjow** [comment, marked as answer]
Hi guys,

To install jpegoptim on Windows for example, simply download here the bin : https://sourceforge.net/projects/jpegoptim/
Then unzip it in a desired folder in your system for example : D:\mySuperBinFolder
Then you can use it in your project like that :

```
$optimizer = new OptimizerChain();
$optimizer->setTimeout(10);
$optimizer->addOptimizer((new Jpegoptim([
    '--all-progressive',
]))->setBinaryPath('D:\mySuperBinFolder'));
```
